setup.py

```diff
@@ -95,6 +104,7 @@ def read(fname):
         'Intended Audience :: Developers',
         'License :: OSI Approved :: MIT License',
         'Operating System :: OS Independent',
+        'Programming Language :: Python :: 2.7',
```

Comment:
We intentionally don't support 2.7 anymore.

Reply:
Is this public? I see no mention of this (important, critical) information in the README.

If true, let's point to the "last version that works with 2.7" in the README and release notes, clearly and prominently.

Reply:
Dropped that classifier.

Reply:
Whatever you guys decide lmk and I can make a change. I did not indent on this PR covering that issue.